src/system-health/health_checker/service_checker.py

```diff
@@ -55,6 +57,11 @@ def __init__(self):
 
         self.load_critical_process_cache()
 
+        self.events_handle = events_init_publisher(EVENTS_PUBLISHER_SOURCE)
+
+    def __del__(self):
```

Comment:
>__del__ [](http://example.com/codeflow?start=8&length=7)

`__del__` is hacky. Suggest follow [https://stackoverflow.com/a/6104568/2514803](https://stackoverflow.com/a/6104568/2514803) for unmanaged resource.

Reply:
No longer use __del__, moved deinit to the end of check() method which is the main driver of the script, used by manager.py